generated/nirfmxbt/nirfmxbt.proto

```diff
@@ -31,27 +325,1873 @@ enum Boolean {
+  nidevice_grpc.Session instrument = 2;
+}
+
+message InitiateRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  string result_name = 3;
+}
+
+message InitiateResponse {
+  int32 status = 1;
+}
+
+message ModAccCfgAveragingRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  oneof averaging_enabled_enum {
+    ModAccAveragingEnabled averaging_enabled = 3;
+    int32 averaging_enabled_raw = 4;
+  }
+  int32 averaging_count = 5;
+}
+
+message ModAccCfgAveragingResponse {
+  int32 status = 1;
+}
+
+message ModAccCfgBurstSynchronizationTypeRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  oneof burst_synchronization_type_enum {
+    ModAccBurstSynchronizationType burst_synchronization_type = 3;
+    int32 burst_synchronization_type_raw = 4;
+  }
+}
+
+message ModAccCfgBurstSynchronizationTypeResponse {
+  int32 status = 1;
+}
+
+message ModAccFetchConstellationTraceRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message ModAccFetchConstellationTraceResponse {
+  int32 status = 1;
+  repeated nidevice_grpc.NIComplexNumberF32 constellation = 2;
+  int32 actual_array_size = 3;
+}
+
+message ModAccFetchDEVMRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message ModAccFetchDEVMResponse {
+  int32 status = 1;
+  double peak_rmsdevm_maximum = 2;
+  double peak_devm_maximum = 3;
+  double ninetynine_percent_devm = 4;
+}
+
+message ModAccFetchDEVMMagnitudeErrorRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message ModAccFetchDEVMMagnitudeErrorResponse {
+  int32 status = 1;
+  double average_rms_magnitude_error_mean = 2;
+  double peak_rms_magnitude_error_maximum = 3;
+}
+
+message ModAccFetchDEVMPerSymbolTraceRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message ModAccFetchDEVMPerSymbolTraceResponse {
+  int32 status = 1;
+  repeated float devm_per_symbol = 2;
+  int32 actual_array_size = 3;
+}
+
+message ModAccFetchDEVMPhaseErrorRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message ModAccFetchDEVMPhaseErrorResponse {
+  int32 status = 1;
+  double average_rms_phase_error_mean = 2;
+  double peak_rms_phase_error_maximum = 3;
+}
+
+message ModAccFetchDemodulatedBitTraceRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message ModAccFetchDemodulatedBitTraceResponse {
+  int32 status = 1;
+  repeated int32 demodulated_bits = 2;
+  int32 actual_array_size = 3;
+}
+
+message ModAccFetchDf1Request {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message ModAccFetchDf1Response {
+  int32 status = 1;
+  double df1avg_maximum = 2;
```

Comment:
Not sure what these are supposed to be our what they started as. `df1_avg`? I guess the param names in the api metadata are `Df1avg` or something. I'd default to keeping it as-is, but we can see if there's any useful context in the .net or LV API.

Reply:
Created [AB#1812677](https://ni.visualstudio.com/DevCentral/_workitems/edit/1812677).